cc/modules/protocol/mpc/snn/src/impl/division.cpp

```diff
@@ -288,7 +288,59 @@ int ReciprocalDiv::ReciprocalDivfor2(
 
     vector<mpc_t> quotient_vec = shared_quotient_vec;
 
-    
+//we should limit the denominator to expand the scope of computation and the accuracy.
+    	vector<mpc_t> SHARED_ONE(vec_size, 0);
+	 if(partyNum == PARTY_A) {
+	SHARED_ONE = vector<mpc_t>(vec_size, FloatToMpcType(1));
+	}
+	vector<mpc_t> SHARED_TEN(vec_size, 0);
+	 if(partyNum == PARTY_A) {
+	SHARED_TEN = vector<mpc_t>(vec_size, FloatToMpcType(10));
+	}
+	vector<mpc_t> SHARED_divTEN(vec_size, 0);
+	if(partyNum == PARTY_A) {
+	SHARED_divTEN = vector<mpc_t>(vec_size, FloatToMpcType(0.1));
+	}
+	vector<mpc_t> judge_val_1(vec_size, 0);
+	vector<mpc_t> judge_val_2(vec_size, 0);
+	vector<mpc_t> judge_val_1_p(vec_size, 0);//msb of judge number
+	vector<mpc_t> judge_val_2_p(vec_size, 0);
+	vector<mpc_t> judge(vec_size, 0);
+	vector<mpc_t> factor(vec_size, 0);//multiple factor
+	vector<mpc_t> denominator_temp(vec_size, 0);
+	vector<mpc_t> numerator_temp(vec_size, 0);
+
+	for(int i=0;i<4;i++)
```

Comment:
add space between `;` and statements, you can refer to other codes.

Reply:
OK ,I have rectified all the similar problem.